experiments/train_resflow_mlp.py

```diff
@@ -107,6 +118,16 @@ def leaky_tanh(x, alpha=1.0, beta=0.1):
 cima_mlp_train = jnp.mean(cima_higher_d_fwd(S_train, jac_mlp))
 cima_mlp_test = jnp.mean(cima_higher_d_fwd(S_test, jac_mlp))
 
+compute_metrics = config['training']['compute_metrics']
```

Comment:
Can you make `compute_metrics` an optional entry for the config file so it is compatible with previous versions, please? You can do it in a similar way as I have done it [here](https://github.com/VincentStimper/resampled-base-flows/blob/b1ea61382f7bd44fa34eecf3c5afc2451ee158ca/larsflow/core.py#L77).

Reply:
Sure! That's much better, I've changed it now